easybuild/easyblocks/l/lammps.py

```diff
@@ -339,9 +513,9 @@ def get_kokkos_arch(cuda_cc, kokkos_arch):
 
         print_msg("Determined cpu arch: %s" % processor_arch)
 
+    gpu_arch = None
     if cuda:
         # CUDA below
-        gpu_arch = None
```

Comment:
In here we should warn that LAMMPS will be built for latest cuda compute specified in `cuda_cc` (that matches a known Kokkos arch) and not all of them. Possibly:
```py
if len(cuda_cc) > 1:
    print_warning("LAMMPS will be built for the latest CUDA compute capability known to Kokkos")
```

Reply:
Is the new warning below enough?

Reply:
Yes, that is fine.